build/datagen-functions/geographies.js

```diff
@@ -0,0 +1,54 @@
+import fs from "fs";
+import stringify from "json-stable-stringify";
+import jsonminify from "jsonminify";
+import path from "path";
+
+import siteConfigData from "../../data/config/site.js";
+
+const siteConfig = siteConfigData.default;
+
+function addLabels(data, labelFunc, labelEsFunc) {
+  data.features.forEach((f) => {
+    // CALLOUT: previously there was a check for "f.label_en" existing, which doesn't seem to be the correct name for the field in the data schema
+    f.properties.label = f.properties.label ?? labelFunc(f.properties.id);
```

Comment:
[Previously](https://github.com/DataWorks-NC/quality-of-life-dashboard/blob/904762e891e6963b8040b721508e0c023e146a3d/build/datagen.js#L68) there was a check for the existence of `f.label_en`, however the right name for the field seems to be `f.label`.